plugins/outputs/stomp/README.md

```diff
@@ -0,0 +1,29 @@
+# STOMP Producer Output Plugin
+
+This plugin writes to a [Active MQ Broker](http://activemq.apache.org/)/
+ for STOMP <http://stomp.github.io>.
+
+It also support Amazon MQ  <https://aws.amazon.com/amazon-mq/>
+
+## CONFIGURATON
+
+```toml
+  ## Host of Active MQ broker
+  host = "localhost:61613"
+
+  ## Queue name for producer messages
+  # queueName = "telegraf"
+
+  ## Username and password if required by the Active MQ server.
+  # username = ""
+  # password = ""
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
+
+  ## Data format to output.
+  # data_format = "json"
```

Comment:
I think you are missing the `toml`-section closing tripple-quotes at the end of the file.